src/sagas/task.saga.js

```diff
@@ -300,12 +305,12 @@ const createStdioChannel = (
       // it will throw
     };
 
-    // TODO: if this channel is ever used with async handlers, make sure to
+    // NOTE: if this channel is ever used with async handlers, make sure to
     // use an expanding buffer in order to avoid losing any information
     // passed up by the child process. Initialize it at a length of 2 because
     // at bare minimum we expect to have 2 messages queued at some point (as
     // the exit channel completes, it should emit the return code of the process
-    // and then immediately END.
+    // and then immediately END.)
```

Comment:
I changed this from a TODO to a NOTE since it isn't really something we plan on doing, it's just helpful context for when we build new things.